byway/src/main/java/com/google/sps/servlets/StopsServlet.java

```diff
@@ -44,40 +43,40 @@
   private final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
   private final Key key = KeyFactory.createKey(Trip.DATASTORE_ENTITY_KIND, 1234);
 
-  /* Passes saved destinations stops (if any) to be shown in the schedule panel */
+  /* Passes saved route to be shown in the schedule panel */
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     List<String> stops = Collections.emptyList();
-    Entity entity;
+    Trip trip;
     try {
-      entity = datastore.get(key);
+      trip = Trip.fromEntity(datastore.get(key));
     } catch (EntityNotFoundException e) {
       logger.atInfo().withCause(e).log(
           "Could not retrieve Entity for Trip with key %s while trying to get the stops", key);
       response.setStatus(HttpServletResponse.SC_NOT_FOUND);
       return;
     }
-    stops = (ArrayList<String>) entity.getProperty("destinations");
+    stops = trip.getRoute();
```

Comment:
Do you need this extra `stops` variable? Seems like you could just use `trip.getRoute()`

Reply:
Yup, I just removed it and changed it to trip, to reflect what I ended up using in my other PR.